BUILD

```
@@ -206,7 +208,7 @@ NODEJS |= {
 
 NODEJS |= {
     "{REGISTRY}/{PROJECT_ID}/nodejs" + version + ":" + tag_base + "-" + arch: "//nodejs:nodejs" + version + label + "_" + user + "_" + arch + "_" + DEFAULT_DISTRO
-    for arch in ARCHITECTURES
+    for arch in node_arch(version)
     for version in NODEJS_VERSIONS
```

One more here:
```suggestion
    for version in NODEJS_VERSIONS
    for arch in node_arch(version)
```

PS. Signed the CLA since it's needed from me 😉